build.gradle

```diff
@@ -714,8 +714,7 @@ uiTestTargets.each { version ->
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
       // TODO (VirtusLab/ide-probe#195): remove the need to set these env variables
-      environment 'IDEPROBE_TEST_SUITE', 'uiTest' + version
-      environment 'IDEPROBE_TEST_CASE', 'artifacts'
+      environment 'MY_CI_SCREENSHOTS_DIR', System.getProperty("user.home") + '/artifacts/uiTest' + version + '/screenshots'
```

Comment:
Hmmm this `MY_` in env var name doesn't sound serious... and `CI_` is also inadequate, as this env var applies on local machines as well 🤔